src/types/common/interfaces/common.interfaces.ts

```diff
@@ -172,12 +172,18 @@ export interface CourseSectionHandlers {
   titleText?: string
 }
 
+declare global {
+  export interface File {
+    src?: string
+  }
+}
```

Comment:
Why do we need this optional `src` property on the global File type, if the File object by default has a `name` property which can be a string representing the file name or the path to the file. [MDN File() constructor documentation](https://developer.mozilla.org/en-US/docs/Web/API/File/File#filename)

Reply:
The optional src property on the global File type is introduced in this code to hold the image's display URL after it has been resized or processed

src property is used for convenience in dealing with image previews or resized images in the application, allowing the file metadata to remain intact while also providing an accessible path for rendering the image